### PR TITLE
Added intro to the README

### PR DESCRIPTION
**Pull request by PeterBell**
Because it was requested in issue #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# ren-shared
\ No newline at end of file
+# Renaissance Shared Project
```
Comment (PeterBell) [inline README.md]: Glad you got the spelling right here!

**Comment by mikepgit** [comment]
👍

**Comment by PeterBell** [comment]
Looks Good To Me (LGTM) :+1:

**Comment by WendyIsenhart** [comment]
:+1

**Comment by CatherineNC** [comment]
🎱  Baa baa black sheep have you any wool?

**Comment by WendyIsenhart** [comment]
:+1:

**Comment by lindsayhaas** [comment]
super

**Comment by jrmcb1943** [comment]
🔢

**Comment by PeterBell** [comment]
Could you let us know more about what this does?

**Comment by heftyt** [comment]
Looks Terrific To Me (LTTM)

**Comment by jamesbolsen** [comment]
ok

**Comment by CatherineNC** [comment]
It's sending us emails live!

**Comment by PeterBell** [comment]
Hey @brntbeer what do you think of this PR? (training, no reply required!)

**Comment by PeterBell** [comment]
There's a typo in the word "about"

**Comment by PeterBell** [comment]
👍

**Comment by kukarzev** [comment]
:+1:

**Pull request merged**